src/YesSql.Provider.MySql/MySqlDialect.cs

```diff
@@ -29,10 +31,61 @@ public class MySqlDialect : BaseDialect
             {DbType.UInt64, "bigint unsigned"},
             {DbType.AnsiStringFixedLength, "char"},
             {DbType.AnsiString, "varchar(127)"},
-            {DbType.StringFixedLength, "varchar"},
+            {DbType.StringFixedLength, "char"},
             {DbType.String, "varchar(255)"},
         };
 
+        static MySqlDialect()
+        {
+            _propertyTypes = new Dictionary<Type, DbType>()
+            {
+                { typeof(object), DbType.Binary },
+                { typeof(byte[]), DbType.Binary },
+                { typeof(string), DbType.String },
+                { typeof(char), DbType.StringFixedLength },
+                { typeof(bool), DbType.Boolean },
+                { typeof(byte), DbType.Byte },
+                { typeof(sbyte), DbType.SByte }, // not supported
+                { typeof(short), DbType.Int16 },
+                { typeof(ushort), DbType.UInt16 }, // not supported
+                { typeof(int), DbType.Int32 },
+                { typeof(uint), DbType.UInt32 },
+                { typeof(long), DbType.Int64 },
+                { typeof(ulong), DbType.UInt64 },
+                { typeof(float), DbType.Single },
+                { typeof(double), DbType.Double },
+                { typeof(decimal), DbType.Decimal },
+                { typeof(DateTime), DbType.DateTime },
+                { typeof(DateTimeOffset), DbType.DateTimeOffset },
+                { typeof(Guid), DbType.Guid },
+                { typeof(TimeSpan), DbType.Int64 },
```

Comment:
shouldnt this be DBType.Time?

Reply:
Tried at first but TimeSpan supports days, not Time in SqlServer.